src/SelfService.Tests/Infrastructure/Api/TestCapabilityCreation.cs

```diff
@@ -6,58 +6,65 @@
 using SelfService.Application;
 using SelfService.Domain.Models;
 using SelfService.Domain.Queries;
+using SelfService.Infrastructure.Persistence.Queries;
 using SelfService.Tests.TestDoubles;
 
-namespace SelfService.Tests.Infrastructure.Api;
+//namespace SelfService.Tests.Infrastructure.Api;
```

Comment:
Please add a comment why we are commenting out this code. If your comment starts with TODO: some IDEs have support for finding these easily.